profiles/common/modules/features/communities/communities.views_default.inc

```diff
@@ -547,7 +547,7 @@ function communities_views_default_views() {
   $handler->display->display_options['pager']['type'] = 'some';
   $handler->display->display_options['pager']['options']['items_per_page'] = '5';
   $handler->display->display_options['pager']['options']['offset'] = '0';
-  $handler->display->display_options['style_plugin'] = 'list';
+  $handler->display->display_options['style_plugin'] = 'ec_europa_link_blocks';
```

Comment:
This style is only available if the module nexteuropa_formatters_views is enabled, but this module is not enable by default.
For ec_resp, we need to keep the style "list" and for ec_europa, we can put this style.